packages/c3/tests/dummy/app/routes/goal-gauge.ts

```diff
@@ -23,7 +27,7 @@ export default class GoalGaugeRoute extends Route {
       dataSource: 'bardOne',
       requestVersion: '2.0',
     });
-    const response = NaviFactResponse.create({ rows: [{ pageViews: 3060000000 }] });
+    const response = new NaviFactResponse(this.clientInjector, { rows: [{ pageViews: 3060000000 }] });
```

Comment:
Did you consider adding the injector to a NaviFactResponse class/subclass, instead of passing it as a constructor arg?